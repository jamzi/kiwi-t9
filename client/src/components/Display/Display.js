import React from 'react';
import { withStyles } from '@material-ui/core/styles';
import Paper from '@material-ui/core/Paper';

const styles = theme => ({
  paper: theme.mixins.gutters({
    paddingTop: 16,
    paddingBottom: 16,
    marginTop: theme.spacing.unit * 3,
    minHeight: '40px',
    display: 'flex',
    flexWrap: 'wrap',
    alignItems: 'center',
    justifyContent: 'center'
  })
});

const Display = props => {
  const { classes, numberString } = props;

  return (
    <Paper className={classes.paper} elevation={4}>
      {numberString}
    </Paper>
  );
};

export default withStyles(styles)(Display);
