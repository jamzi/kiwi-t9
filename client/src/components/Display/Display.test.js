import React from 'react';
import ReactDOM from 'react-dom';
import Display from './Display';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer.create(<Display numberString="test" />).toJSON();
  expect(tree).toMatchSnapshot();
});
