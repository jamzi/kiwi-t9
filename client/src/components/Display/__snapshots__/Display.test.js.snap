// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="MuiPaper-root-2 MuiPaper-elevation4-8 MuiPaper-rounded-3 Display-paper-1"
>
  test
</div>
`;
